client/jetpack-connect/signup.js

```diff
@@ -85,10 +85,16 @@ export class JetpackSignup extends Component {
 		} );
 	}
 
+	isWoo( props = this.props ) {
```

Comment:
Could we just use `const { authQuery } = this.props;` inside the method instead? It doesn't seem like we're using this as an argument anywhere.

Reply:
Handled this and the other piece of feedback in https://github.com/Automattic/wp-calypso/pull/32993/commits/260bfd5c573b6916d8db91ba24ca33bea66dd254.